Pear/View/OnboardingSearchDropdownView.swift

```diff
@@ -31,26 +33,40 @@ class OnboardingSelectTableView: UITableView {
     }
 }
 
+enum OnboardingSearchDropdownType {
+    case places // search from Google Places API
+    case local // search locally
+}
+
 class OnboardingSearchDropdownView: UIView {
 
     // MARK: - Private View Vars
     private let searchBar = UISearchBar()
     private let tableView = OnboardingSelectTableView()
+    private var placesClient: GMSPlacesClient!
 
     // MARK: - Private Data Vars
     private weak var delegate: OnboardingDropdownViewDelegate?
     private var placeholder: String
     private var resultsTableData: [String] = []
+    private let searchType: OnboardingSearchDropdownType
     private var tableData: [String]
 
     // MARK: - Private Constants
     private let fieldsCornerRadius: CGFloat = 8
 
-    init(delegate: OnboardingDropdownViewDelegate, placeholder: String, tableData: [String]) {
+    init(
+        delegate: OnboardingDropdownViewDelegate,
+        placeholder: String,
+        tableData: [String],
+        searchType: OnboardingSearchDropdownType
+    ) {
         self.delegate = delegate
         self.placeholder = placeholder
         self.tableData = tableData
+        self.searchType = searchType
         super.init(frame: .zero)
+        placesClient = GMSPlacesClient.shared()
```

Comment:
Put this before init and make `placesClient` non-optional. 

Could you even just say `let placesClient = GMSPlacesClient.shared()` right in the class body?

Edit: just delete this. See my comment below.